stl/inc/format

```diff
@@ -117,24 +117,25 @@ public:
     using const_iterator = typename basic_string_view<_CharT>::const_iterator;
     using iterator       = const_iterator;
 
-    constexpr explicit basic_format_parse_context(basic_string_view<_CharT> _Fmt, size_t _Num_args_ = 0) noexcept

```

Comment:
I tried to plumb `locale`s into `basic_format_parse_context` and friends, which necessitated removing ton of `constexpr`. Since I decided that route was a dead end and yanked much of it back it, I can restore a lot of `constexpr` here.